helm-charts/submarine/Chart.yaml

```diff
@@ -23,6 +23,9 @@ version: 0.8.0-SNAPSHOT
 icon: https://submarine.apache.org/assets/themes/submarine/img/submarine_white_logo.png
 dependencies:
   - name: training-operator
-    version: "0.1.0"
+    version: "0.7.0"
   - name: notebook-controller
     version: "0.1.0"
+  - name: seldon-core-operator
+    version: "1.10.0"
+    repository: https://storage.googleapis.com/seldon-charts
```

Comment:
If the user already has installed `seldon`, I think we can add a condition to make helm not install this dependency. For example:
```yaml
condition: seldon-core-operator.enabled
```